src/expression.rs

```diff
@@ -57,8 +57,8 @@ impl PyExpr {
         expr.into()
     }
 
-    fn __str__(&self) -> PyResult<String> {
```

Comment:
Removed since redundant if implemented the same way, e.g. `str(literal(5))` will fall back to `__repr__` if `__str__` is not implemented)